src/xcode/ENA/ENA/Source/Scenes/Home/HomeViewController.swift

```diff
@@ -104,6 +106,52 @@ final class HomeViewController: UIViewController {
 		updateBackgroundColor()
 	}
 
+	/// This method sets up a background fetch alert, and presents it, if needed.
+	/// Check the `createBackgroundFetchAlert` method for more information.
+	func setupBackgroundFetchAlert() {
+		guard let alert = createBackgroundFetchAlert(
+			status: UIApplication.shared.backgroundRefreshStatus,
+			inLowPowerMode: ProcessInfo.processInfo.isLowPowerModeEnabled,
+			hasSeenAlertBefore: homeInteractor.store.hasSeenBackgroundFetchAlert
+			) else { return }
+
+		self.present(
+			alert,
+			animated: true,
+			completion: nil
+		)
+	}
+
+	/// This method checks whether the below conditions in regards to background fetching have been met
+	/// and creates the corresponding alert.
+	/// The error alert should only be shown:
+	/// - once
+	/// - if the background refresh is disabled
+	/// - if the user is __not__ in power saving mode, because in this case the background
+	///   refresh is disabled automatically. Therefore we have to explicitly check this.
+	func createBackgroundFetchAlert(
```

Comment:
`private` is better?

Reply:
I can make `setupBackgroundFetchAlert` private, that is no problem. 
However, if I make `createBackgroundFetchAlert()` private, I am no longer able to test it. And if I am not able to test this method, I can barely test anything else in this PR, which would result in low test coverage and I want to avoid that.